src/server/pom.xml

```diff
@@ -88,11 +90,6 @@
             <artifactId>dropwizard-client</artifactId>
```

Comment:
Dropwizard makes a heavy usage of Jackson library – another frequent source of CVEs. Dropwizard 2.0.25 depends on Jackson 2.10.3 which is already old. I strongly recommend forcing Jackson version to the latest: 2.13.0.

This can be done by adding the following snippet to this pom:

```
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>com.fasterxml.jackson</groupId>
                <artifactId>jackson-bom</artifactId>
                <version>2.13.0</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
```

Reply:
Unfortunately upgrading to 2.13.0 won't work out of the box. The following error pops up in acceptance tests:

```
 java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Joda date/time type `org.joda.time.DateTime` not supported by default: add Module "com.fasterxml.jackson.datatype:jackson-datatype-joda" to enable handling
```

I guess it's safer to stay on 2.10.5. Luckily enough, it seems there is currently no [CVE targeting that version](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=jackson).